src/compass/utils/lut.py

```diff
@@ -0,0 +1,22 @@
+def compute_lut(burst, xstep=500, ystep=50):
```

Comment:
```suggestion
def compute_geocoding_correction_lut(burst, xstep=500, ystep=50):
```
Verbosity for clarity.